devtools-ci-index.yaml

```diff
@@ -898,6 +898,12 @@
             timeout: '30m'
             svc_name: keycloak-server
         - '{ci_project}-{git_repo}':
+            git_organization: almighty
+            git_repo: keycloak
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
```

Comment:
It failed before because we were building 3.0.0.CR1-SNAPSHOT and maven decencies were not published for those versions. But now we are depending on 3.0.0.Final artifacts. So, it should work. Needs to be tested though.